.pre-commit-config.yaml

```diff
@@ -4,10 +4,13 @@ repos:
   hooks:
   - id: check-merge-conflict
   - id: check-toml
+    exclude: (/.*)?templates(/.*)?$
   - id: check-yaml
+    exclude: (/.*)?templates(/.*)?$
   - id: end-of-file-fixer
     exclude_types: [markdown]  # incompatible with Literate.jl
-- repo: https://github.com/qiaojunfeng/pre-commit-julia-format
-  rev: v0.2.0
+
+- repo: "https://github.com/domluna/JuliaFormatter.jl"
+  rev: v2.0.0  
```

Comment:
Does this mean it will use JuliaFormatter.jl v2? That release was buggy and got yanked.